src/widgets/paragraph.rs

```diff
@@ -402,6 +403,7 @@ impl<'a> Styled for Paragraph<'a> {
 
 #[cfg(test)]
 mod test {
+
```

Comment:
Why is `clippy fmt` happy about this? :eyes: 
I would have expected that fmt removes this empty line at the start of a block.

Reply:
Ha, I initially wrote a test using rstest for this and vscode inserted an import automatically. Decided not to use it and missed that there was an added newline. It can be removed.

Reply:
Best suggestion ever I guess? :ghost: 

```suggestion
```